src/scripts/main.js

```diff
@@ -1,3 +1,20 @@
 'use strict';
 
-// write your code here
+let total = 0;
+let average = 0;
+
+const numbers = [...document.querySelectorAll('span.population')];
+
+const population = numbers.map((item) => item.textContent.replaceAll(',', ''));
```

Comment:
Consider handling cases where the textContent might not be a valid number after removing commas. This could lead to NaN values in the 'population' array.